drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -1224,7 +1224,10 @@ static void vc4_hdmi_encoder_post_crtc_powerdown(struct drm_encoder *encoder,
 	if (vc4_hdmi->variant->phy_disable)
 		vc4_hdmi->variant->phy_disable(vc4_hdmi);
 
+	/* we no longer require a minimum clock rate */
+	clk_set_min_rate(vc4_hdmi->pixel_bvb_clock, 0);
 	clk_disable_unprepare(vc4_hdmi->pixel_bvb_clock);
+	clk_set_min_rate(vc4_hdmi->hsm_clock, 0);
```

Comment:
Can you explain why the pairing of this line with the following line (which refence `hsm_clock` and `pixel_clock`) doesn't match other `clk_set_min_rate` and `clk_disable_unprepare` pairs - or at least say that it is expected and correct?

Reply:
Good point. `vc4_hdmi->hsm_clock` and `vc4_hdmi->pixel_bvb_clock` are the clocks that previously were used with `clk_set_min_rate`, so the lines added are as intended.

But, when I moved them from `disable` to `destroy` I assumed they lined up with the clocks being disabled, but as you say that is not quite matching. Let me have a closer look.

Reply:
Thanks - I can believe that does what you want.